webapp/javascript/ui/StatusMessage/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import cx from 'classnames';
+import styles from './StatusMessage.module.css';
+
+interface StatusMessageProps {
+  type: 'error' | 'success';
+  children: string;
+}
+
+export default function StatusMessage({ type, children }: StatusMessageProps) {
+  return children ? (
+    <div
+      className={cx({
+        [styles.statusMessage]: true,
+        [styles.error]: type === 'error',
+        [styles.success]: type === 'success',
```

Comment:
This kinda implies it's possible to have both `styles.error` and `styles.success`, which is obviously untrue.

Can we rewrite as a pure function?
```js
const  getClassnameForType = () => {
  switch (type) {
    case 'error': return styles.error;
    case 'success': return styles.success;
  }
}
```